syntax = "proto3";

message ProtoConfig {
  string repo_path = 1;
  string api_address = 2;
  string dht_address = 3;
  string bootstrap_node_address = 4;
  string gc_size = 5;
  string gc_time = 6;
}
